pgtap/allpairs/floydWarshall-rows-check.sql

```diff
@@ -0,0 +1,87 @@
+\i setup.sql
+
+SELECT plan(20);
+
+SET extra_float_digits = -3;
+
+-- Check whether the same set of rows are returned always
+
+PREPARE expectedOutputDirected AS
+SELECT * FROM pgr_floydWarshall(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id'
+);
+
+PREPARE descendingOrderDirected AS
+SELECT * FROM pgr_floydWarshall(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id DESC'
+);
+
+PREPARE randomOrderDirected AS
+SELECT * FROM pgr_floydWarshall(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()'
+);
+
+PREPARE expectedOutputUndirected AS
+SELECT * FROM pgr_floydWarshall(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id',
+    directed => false
+);
+
+PREPARE descendingOrderUndirected AS
+SELECT * FROM pgr_floydWarshall(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY id DESC',
+    directed => false
+);
+
+PREPARE randomOrderUndirected AS
+SELECT * FROM pgr_floydWarshall(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()',
+    directed => false
+);
+
+SELECT SETSEED(1);
```

Comment:
Actually this line is not needed. because you are not comparing printed characters that one would be necessary on the doc-queries files but those are not testing files.

Reply:
You mean `SELECT SETSEED(1);`?
Well, yeah, it isn't needed. I added it so that any particular test can be regenerated whenever required, hence we can find out a possible source of error by running the same test locally anytime.